cli/src/main/java/org/crsh/cli/impl/lang/Util.java

```diff
@@ -27,7 +27,11 @@
 import java.util.regex.Matcher;
 import java.util.regex.Pattern;
 
-public class Util {
+public final class Util {
+	
+  private Util(){
+	  throw new AssertionError("Must not instantiate this class");
```

Comment:
If this is really necessary, perhaps it should throw an IllegalStateException, instead.